sling/nlp/wikicat/generator.py

```diff
@@ -294,8 +296,32 @@ def post_process(self, parses):
   # Returns category members that are not categories themselves.
   def get_members(self, frame):
     output = []
+    allowed = []
+    store = frame.store()
+    if self.h_category_contains in frame:
```

Comment:
Minor thing. Dealing with the entire case of `if self.h_category_contains not in frame:` first (including looping over members and a `return output` at the end) might be easier to read albeit slightly more code.

Reply:
Good suggestion. Done.